File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int current = 0;
        for(int i = 0 ; i < nums.size() ; i++){
            if(nums[current]!=nums[i]){
                current++;
                nums[current] = nums[i];
            }
        }
            return current+1;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int current = nums[0];
        int pos = 1;
        for(int i = 1 ; i < nums.length ; i++){
            if(nums[i]!=current){
                current = nums[i];
                nums[pos] = nums[i];
                pos++;
            }   
        }
        return pos;
    }
}